source/common/upstream/cds_api_impl.cc

```diff
@@ -30,11 +30,14 @@ CdsApiImpl::CdsApiImpl(const envoy::api::v2::core::ConfigSource& cds_config, Clu
     : cm_(cm), scope_(scope.createScope("cluster_manager.cds.")) {
   Config::Utility::checkLocalInfo("cds", local_info);
 
+  bool is_delta = (cds_config.api_config_source().api_type() ==
```

Comment:
Nit: `const` for both of these local vars.

Reply:
Done.